session/session.go

```diff
@@ -1384,6 +1385,13 @@ func (s *session) ParseWithParams(ctx context.Context, sql string, args ...inter
 	for _, warn := range warns {
 		s.sessionVars.StmtCtx.AppendWarning(util.SyntaxWarn(warn))
 	}
+	if variable.TopSQLEnabled() {
+		normalized, digest := parser.NormalizeDigest(sql)
+		if digest != nil {
+			// Fixme: reset/clean the label when sql execute finish.
```

Comment:
Is this FIXME fixed?

Reply:
No, some other internal SQL still need fix it.